iambic/plugins/v0_1_0/aws/organizations/scp/models.py

```diff
@@ -0,0 +1,582 @@
+from iambic.core.models import (
+    AccountChangeDetails,
+    BaseModel,
+    ExpiryModel,
+    ProposedChange,
+    ProposedChangeType,
+)
+from iambic.core.utils import normalize_dict_keys, plugin_apply_wrapper
+from iambic.plugins.v0_1_0.aws.models import (
+    AccessModel,
+    AWSAccount,
+    AWSTemplate,
+    Description,
+    Tag,
+)
+from iambic.plugins.v0_1_0.aws.organizations.scp.utils import (
+    apply_update_policy,
+    apply_update_policy_tags,
+    apply_update_policy_targets,
+    create_policy,
+    delete_policy,
+    get_policy,
+    service_control_policy_is_enabled,
+)
+
+if TYPE_CHECKING:
+    from iambic.plugins.v0_1_0.aws.iam.models import Path
+    from iambic.plugins.v0_1_0.aws.iambic_plugin import AWSConfig
+    from iambic.plugins.v0_1_0.aws.models import AWSOrganization
+
+
+AWS_SCP_POLICY_TEMPLATE = "NOQ::AWS::Organizations::SCP"
+
+
+class StatementEffect(str, Enum):
+    ALLOW = "Allow"
+    DENY = "Deny"
+
+
+class ServiceControlPolicyTargetItemType(str, Enum):
+    ACCOUNT = "ACCOUNT"
+    ORGANIZATIONAL_UNIT = "ORGANIZATIONAL_UNIT"
+    ROOT = "ROOT"
+
+
+class ServiceControlPolicyItemType(str, Enum):
+    SERVICE_CONTROL_POLICY = "SERVICE_CONTROL_POLICY"
+    TAG_POLICY = "TAG_POLICY"
+    BACKUP_POLICY = "BACKUP_POLICY"
+    AISERVICES_OPT_OUT_POLICY = "AISERVICES_OPT_OUT_POLICY"
+
+
+class OutputModels(PydanticBaseModel):
+    """Output models from AWS Organizations client"""
+
+    class Config:
+        use_enum_values = True
+
+
+class PolicyStatement(AccessModel, ExpiryModel):
+    """
+
+    Note:
+        Unsupported elements:
+            - Principal
+            - NotPrincipal
+            - NotResource
+    """
+
+    effect: str = Field(..., description="Allow | Deny")
+    action: Optional[Union[List[str], str]] = Field(
+        None,
+        description="A single regex or list of regexes. "
+        "Values are the actions that can be performed on the resources in the policy statement",
+        example="dynamodb:list*",
+    )
+    not_action: Optional[Union[List[str], str]] = Field(
+        None,
+        description="An advanced policy element that explicitly matches everything except the specified list of actions."
+        "DON'T use this with effect: allow in the same statement OR policy",
+    )
+    resource: Optional[Union[List[str], str]] = Field(
+        None,
+        description="A single regex or list of regexes. Values specified are the resources the statement applies to",
+    )
+    condition: Optional[dict] = Field(
+        None,
+        description="An optional set of conditions to determine of the policy applies to a resource.",
+    )
+    sid: Optional[str] = Field(
+        None,
+        description="The Policy Statement ID.",
+    )
+
+    @property
+    def resource_type(self):
+        return "aws:organizations:service_control_policy"
+
+    @property
+    def resource_id(self):
+        return self.sid
+
+
+class PolicyDocument(AccessModel, ExpiryModel):
```

Comment:
This one is covered by https://github.com/noqdev/iambic/blob/508af0bd3665e65e7a300f463794a70e51b1ef0a/iambic/plugins/v0_1_0/aws/iam/policy/models.py#L195, can we reuse

Reply:
Same as https://github.com/noqdev/iambic/pull/384#discussion_r1199488869